src/aspire/utils/misc.py

```diff
@@ -48,6 +50,31 @@ def importlib_path(package, resource):
     return p
 
 
+def timestamp_filename(filepath, move=True):
```

Comment:
Also, we never call it with `move=False` (except for in testing), so perhaps it's not necessary to have this as an argument?

Reply:
We do use `move=False` in `ImageSource.save()`. In this case we let `RelionSource` rewrite the old star file with the new timestamped name. This handles rewriting the starfile (with updated mrcs filenames) as well as the corresponding mrcs files.

We could still remove this argument and then just let `RelionSource` overwrite the file that `timestamp_filename` has renamed. Let me know what you think.

Reply:
Ok I missed that. Then let's leave this as is.